### ISSUE TITLE
PHP Fatal error:  Declaration of Codedungeon\PHPUnitPrettyResultPrinter\Printer::writeProgress($progress) must be compatible with PHPUnit\TextUI\ResultPrinter::writeProgress($progress): void in /home/vagrant/mailsoft/vendor/codedungeon/phpunit-result-printer/src/Printer.php on line 36

### ISSUE THREAD

**Issue by t202wes** [mikeerickson/phpunit-pretty-result-printer#45]
```
PHP Fatal error:  Declaration of Codedungeon\PHPUnitPrettyResultPrinter\Printer::writeProgress($progress) must be compatible with PHPUnit\TextUI\ResultPrinter::writeProgress($progress): void in /home/vagrant/mailsoft/vendor/codedungeon/phpunit-result-printer/src/Printer.php on line 36
```

**Comment by mikeerickson** [comment]
I am aware of the issue and have a fix ready which will fix the issue.  An update will be posted later this afternoon

**Comment by mikeerickson** [comment]
Duplicate of #43

**Comment by mikeerickson** [comment]
Fixed in `0.9.0` release
